docs/koji.md

```diff
@@ -23,6 +23,7 @@ For atomic-reactor container image builds the `image` type is used, and so type-
 Data which is placed here includes:
 
 - `build.extra.image.autorebuild` (boolean): true if this build was triggered automatically; false otherwise
+- `build.extra.image.help` (string or null): filename if this build has a markdown help converted; null otherwise
```

Comment:
Does "filename of the markdown help file in the repository if this build has a markdown help converted to man page; null otherwise" sound good?